flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/FlinkDistribution.java

```diff
@@ -86,7 +84,7 @@
 
 	private Configuration defaultConfig;
 
-	public FlinkDistribution() {
+	FlinkDistribution() {
```

Comment:
other non-overridden methods can also be package private?

Reply:
This represents the actual API that would be public if it were in another package; will not change it.